### PR TITLE
Converting google doc draft into acsiidoc 

### PR DESCRIPTION
**Pull request by g-murray**
[AAP-29834](https://issues.redhat.com/browse/AAP-29834)

Inputting the content from the Google doc draft into the corresponding assembly and modules.

### CODE REVIEW

File: stories/topics/con-saas-backup-and-diaster-recovery.adoc [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
+|===
+| Component   | Snapshot Frequency | Retention Policy
+| Database    | Daily              | 7 days
+| File System | Daily              | 7 days
+|===
+
+This recovery data is used in the event that it becomes unlikely that an AWS regional outage will be recoverable within a reasonable period. 
+In the event of such a circumstance, the following process may be followed to recover an {PlatformNameShort} deployment in a different AWS region:
+
+* *Add Steps*
```
Comment (g-murray) [inline stories/topics/con-saas-backup-and-diaster-recovery.adoc:15]: Waiting on SME input - will circle back for this

File: stories/topics/con-saas-sre-access-and-management.adoc [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+Site Reliability Engineering (SRE) access is specific to the infrastructure and services running {PlatformNameShort}. 
+Red Hat does not access the {PlatformNameShort} interfaces or APIs except for extenuating circumstances, such as a support engagement that requires it. 
+In such cases, access would have to be provided by the customer to the SRE team for troubleshooting those issues.
+
+SRE access to control plane resources is limited to operations that must be performed by a person and cannot be automated. 
+In such cases, any access to a particular control plane follows a request-and-approval process, and the access is audited, to ensure that only permitted personnel have access to perform these operations.
+
+The following describes the process that SREs use to access resources and the audit data that is captured whenever an operation against resources is performed.
+
+* *need steps*
```
Comment (g-murray) [inline stories/topics/con-saas-sre-access-and-management.adoc:13]: Waiting on SME input - will circle back for this

File: stories/topics/con-saas-application-monitoring-and-customer-audits.adoc [hunk 3]
```diff
@@ -1,2 +1,5 @@
 [id="application-monitoring-and-customer-audits"]
-= Application-monitoring-and-customer-audits
\ No newline at end of file
+= Application monitoring and customer audits
+
+The {PlatformNameShort} activity stream provides detailed information about access to {PlatformNameShort} and how you can use it. 
+If you need to retain this information for auditing or compliance reasons, we encourage you to export logs to supported logging services to retain and query logs.
```
Comment (ianf77) [inline stories/topics/con-saas-application-monitoring-and-customer-audits.adoc:5]: If you retain this information... you must export...

File: stories/topics/con-saas-connectivity.adoc [hunk 4]
```diff
@@ -1,3 +1,12 @@
 [id="con-saas-connectivity"]
 
-= Connectivity
\ No newline at end of file
+= Connectivity
+
+In general, the execution plane should be able to communicate to the control plane when the following conditions are met:
```
Comment (ianf77) [inline stories/topics/con-saas-connectivity.adoc:5]: is able to communicate

File: stories/topics/con-saas-connectivity.adoc [hunk 5]
```diff
@@ -1,3 +1,12 @@
...
+
+* Polling (mesh-ingress): Execution nodes must be able to route stateful egress traffic to the domain of the {PlatformNameShort} deployment over port 443.
+* Push: A configurable firewall port must be opened within a customer’s remote networks to allow for {PlatformNameShort} to push information to execution nodes.
+
+You can configure automation mesh nodes behind firewalls, proxy servers, and similar services. 
+These services need to route or proxy traffic originating from {PlatformNameShort} without altering headers, payload, or other information that would affect functionality of the automation mesh.
```
Comment (ianf77) [inline stories/topics/con-saas-connectivity.adoc:11]: These services route (they don't _need_ to)

File: stories/topics/con-saas-networking-with-overlapping-CDIR-blocks.adoc [hunk 6]
```diff
@@ -1,3 +1,7 @@
 [id="con-saas-networking-with-overlapping-CDIR-blocks"]
 
-= Networking with overlapping CDIR blocks
\ No newline at end of file
+= Networking with overlapping CDIR blocks
+
+Automation mesh enables connectivity between the control plane and multiple networks that use the same Classless Inter-Domain Routing (CIDR) block (that is, the same class A address space repeated across different clouds or data centers). 
+Execution nodes treat the network in which they are deployed as the local network. 
+You need at least one execution node instance paired with an instance group to target automation in each of the networks.
```
Comment (ianf77) [inline stories/topics/con-saas-networking-with-overlapping-CDIR-blocks.adoc:7]: You require or You must have

File: stories/topics/con-saas-shape.adoc [hunk 7]
```diff
@@ -1,3 +1,24 @@
 [id="con-saas-shape"]
 
-= Shape
\ No newline at end of file
+= Shape
+
+Your execution plane will vary in size and shape based on the type of automation that is run and the locations that the mesh needs to connect to.
```
Comment (ianf77) [inline stories/topics/con-saas-shape.adoc:5]: plane varies in size

that the mesh connects to.

File: stories/topics/con-saas-shape.adoc [hunk 8]
```diff
@@ -1,3 +1,24 @@
...
+= Shape
+
+Your execution plane will vary in size and shape based on the type of automation that is run and the locations that the mesh needs to connect to. 
+Use the following to help guide your automation mesh implementation model:
+
+* {PlatformNameShort} minimum requirements:
+** Hop Nodes: Hop nodes are only required when not using mesh ingress. 
+Hop nodes typically require minimum resources. 
+The shape of a hop node depends on the number of execution nodes connected to it. 
+Typically a virtual machine (VM) with 2 vCPUs and 2 Gb RAM can route traffic for 2-4 execution nodes.
```
Comment (ianf77) [inline stories/topics/con-saas-shape.adoc:12]: Are you sure Gb is the correct abbreviation, not GB?

File: stories/topics/con-saas-shape.adoc [hunk 9]
```diff
@@ -1,3 +1,24 @@
...
+The shape of a hop node depends on the number of execution nodes connected to it. 
+Typically a virtual machine (VM) with 2 vCPUs and 2 Gb RAM can route traffic for 2-4 execution nodes.   
+** Execution Nodes: {PlatformNameShort} minimum requirements are 4 vCPUs and 8 Gb RAM for execution nodes. 
+** {PlatformNameShort} performance and scale testing showed that 2 m4.xlarge execution nodes could effectively automate against 200 managed active nodes constantly with enough remaining CPU and RAM as a buffer to prevent job failures in the event of a spike on those machines. 
+You can use this as a baseline to consider the right machine shapes for your environment when dealing with a CPU-heavy consistent automation load.
+You must monitor and tune the execution plane based on the needs of your organization’s automation.
+* If you have automation in fewer locations (such as geographies or other clouds) must create a mesh with fewer VMs that can be scaled vertically. 
+Most clouds and hypervisors enable the ability to change shapes with minimal downtime.
+* If you have CPU or RAM intensive automation, you will require larger machine shapes.
```
Comment (ianf77) [inline stories/topics/con-saas-shape.adoc:19]: you require

File: stories/topics/con-saas-shape.adoc [hunk 10]
```diff
@@ -1,3 +1,24 @@
...
+* If you have automation that spans multiple locations (such as geographies or other clouds), you should create a mesh with nodes that enable connectivity to those locations.
```
Comment (ianf77) [inline stories/topics/con-saas-shape.adoc:20]: clouds), create a mesh...

File: stories/topics/con-saas-shape.adoc [hunk 11]
```diff
@@ -1,3 +1,24 @@
...
+* You can consider the use of different CPU architectures, like ARM, and reserved instances to reduce the cost of the execution plane.
+* If you are interested in configuring redundancy in the automation mesh, you  should configure at least 2 mesh nodes of the same shape in the same region, but in different availability zones, with each machine connected to both mesh ingress hop nodes.
```
Comment (ianf77) [inline stories/topics/con-saas-shape.adoc:22]: two

File: stories/topics/con-saas-sre-access-and-management.adoc [hunk 12]
```diff
@@ -1,2 +1,13 @@
 [id="con-saas-sre-access-and-management"]
-= SRE access and management
\ No newline at end of file
+= SRE access and management
+
+Site Reliability Engineering (SRE) access is specific to the infrastructure and services running {PlatformNameShort}. 
+Red Hat does not access the {PlatformNameShort} interfaces or APIs except for extenuating circumstances, such as a support engagement that requires it. 
+In such cases, access would have to be provided by the customer to the SRE team for troubleshooting those issues.
```
Comment (ianf77) [inline stories/topics/con-saas-sre-access-and-management.adoc:6]: access is provided by

**Review by ianf77 — COMMENTED** [review]
A few comments

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**